packages/frontend/editor-ui/src/app/constants/experiments.ts

```diff
@@ -99,6 +99,12 @@ export const AI_BUILDER_TEMPLATE_EXAMPLES_EXPERIMENT = {
 	variant: 'variant',
 };
 
+export const AI_BUILDER_MULTI_AGENT_EXPERIMENT = {
```

Comment:
There was a discussion yesterday that we should create the feature flag in Posthog before we add it to the code to avoid number duplication - I don't see the feature flag on PH yet so need to put it in there!